charts/ks-devops/values.yaml

```diff
@@ -12,8 +12,25 @@ image:
   tools:
     repository: devops-tools
     tag: master
+  gc:
+    repository: devops-gc
+    tag: master
   pullPolicy: IfNotPresent
 
+gcJobs:
```

Comment:
Does we need to add descriptions about GC in the `README.md`?

Reply:
Yes